### PR TITLE
Databases week2

### PR DESCRIPTION
**Pull request by Divyajg**

### CODE REVIEW

File: databases/week2/db-week2.sql [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+#Tasks of users with email ends in @spotify.com
+SELECT user.id as user_id,  task.title as task, user.email as "email_@spotify.com"
+FROM user, user_task, task WHERE 
+user.email LIKE "%@spotify.com" AND
+user_task.task_id=task.id AND 
+user.id=user_task.user_id;
+
+--Get all the tasks for 'Donald Duck' with status 'Not started'
+# Tasks for 'Donald Duck' with status 'Not started'
+SELECT task.title as Task, task.description as Description, status.name as Status
```
Comment (Elia-Git) [inline databases/week2/db-week2.sql:63]: how do I know if this task belongs to 
hint, you can add the name column like
 `SELECT user.name,task.title as Task, task.description as Description, status.name as Status`
.....

File: databases/week2/db-week2.sql [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+task.status_id=status.id AND
+user.id=user_task.user_id AND
+user_task.task_id=task.id;
+
+--Get all the tasks for 'Maryrose Meadows' that were created in september (hint: month(created)=month_number)
+#tasks for 'Maryrose Meadows' that were created in september
+SELECT * FROM user_task
+JOIN task ON task_id=task.id
+JOIN user ON user_id=user.id
+WHERE user.name='Maryrose Meadows' AND month(task.created) = 09;
```
Comment (Elia-Git) [inline databases/week2/db-week2.sql:76]: Remember to Use "*"  where it makes sense..
here you can select some specific columns instead ,,like this 
```
SELECT user.name, task.title, task.created
FROM ....
```
so the desired table might looks much better

**Review by Elia-Git — COMMENTED** [review]
Good in general. Pls react to given comments and you can ship it afterwards.

**Review by Elia-Git — APPROVED** [review]
Good job,also the ERD!

File: databases/week2/db-week2.sql (hunk 1 above)
Comment (Divyajg) [reply databases/week2/db-week2.sql:63]: Okay. I might have added the user.name as well.
Thank you.

File: databases/week2/db-week2.sql (hunk 2 above)
Comment (Divyajg) [reply databases/week2/db-week2.sql:76]: Yes, noted. Just like I
 specified for the above task!

**Pull request merged**